lxd/db/cluster/cluster_links.go

```diff
@@ -91,3 +93,102 @@ func (c ClusterLinkType) Value() (driver.Value, error) {
 
 	return nil, fmt.Errorf("Invalid cluster link type %q", c)
 }
+
+// CreateClusterLinkConfig creates config for a new cluster link with the given name.
+func CreateClusterLinkConfig(ctx context.Context, tx *sql.Tx, name string, config map[string]string) error {
```

Comment:
any reason this isnt using a DB generated function like `CreateInstanceConfig` does?